account_credit_control/README.rst

```diff
@@ -16,14 +21,24 @@ You are able to specify a particular policy for one partner or one invoice.
 Usage
 -----
 
-Menu entries are located in ``Accounting > Periodical Processing > Credit
-Control``.
+Menu entries are located in ``Accounting > Report > Credit Control``.
```

Comment:
I don't undersand, the name of the main menu is "Accounting". I don't see the menu "Invoicing"